doc/devguide/extending/app-layer/img/DnsRequestUnidirectionalTransaction.msc

```diff
@@ -0,0 +1,13 @@
+# MSC Sequence Diagram Example: DNS Query Transaction
+
+msc {
+    # Chart Options
+    arcgradient = "10";
+
+    # Entities
+    a [ label = "Client" ], b [ label = "Server" ];
+
+    # Message Flow
+    a =>> b [ label = "DNS Request" ];
+    --- [ label = "Transaction Completed" ];
+}
```

Comment:
I'd go one more step and add the DNS Response, transaction complete.  Just to complete the life cycle of a DNS request for an end-user.